deploy/helm/templates/addons/csi-s3-addon.yaml

```diff
@@ -27,6 +27,12 @@ spec:
         - name: {{ include "kubeblocks.fullname" . }}-cloud-provider
           key: csi-s3
 
+      {{- $cloudProvider := (include "kubeblocks.cloudProvider" .) }}
+      {{- if eq $cloudProvider "huaweiCloud" }}
+      setValues:
+        - kubeletPath=/mnt/paas/kubernetes/kubelet
+      {{- end }}
+
```

Comment:
Like `snapshot-controller`, use `defaultInstallValues.selectors` to set its install value based on `KubeProvider`.

https://github.com/apecloud/kubeblocks/blob/ac647917fc9b28f03ba97a23e3693e156240907f/deploy/helm/templates/addons/snapshot-controller-addon.yaml#L54-L82

Reply:
I don't quite understand how to do this, defaultInstallValues (or more precisely AddonInstallSpecItem within it) only allows us to set some special fields, but in this case we need to set a custom value.

https://github.com/apecloud/kubeblocks/blob/2c4e6c4af2216d089d6e2896d9768fe5c2be8e9f/apis/extensions/v1alpha1/addon_types.go#L358-L378